README.md

```diff
@@ -6,6 +6,27 @@ The Bosch Connected Devices Development Kit (CDDK) has been originally used to d
 
 Kiso (Japanese for "basis") is an Eclipse Foundation project that is using the original code base of CDDK. Kiso is transferring the Bosch proprietary licence code piece-by-piece to EPLv2 open source. Kiso's quality goals - understandability, usability, product quality, extensibility and performance - are the key factors that enable a fast prototype and product development. By this, it reduces the time to market together with minimizing technical debts for almost all kinds of IoT "Things".
 
+## Scope ##
```

Comment:
As Peter i would never find the Quickstart now, as i do not read anything about the Personas.
Should we change the order?

Reply:
Good point. I added a table of content. Do you think this resolves the huge text block?

Reply:
Is it necessary to put the Personas in the main Readme? To my knowledge personas are used to identify requirements and are part of the contribution process. Doesn't sound like it'd too interesting for plain users/non-contributors.

I propose a Hugo page somewhere in the Contribution section as a better location to store these.

Reply:
I like the idea of being able to see what this project is actually all about. Also, this is meant to be a guideline for all of us in our daily decision process (why are we doing what we're doing right now and how does is benefit Kiso's use cases?). I would like to leave it in the README to have it visible on the first project contact. But I aggree tl;dr -> what about moving the section down, below the resources? Or, alternatively, what about linking it as an item in the resources? Please comment @elbosch and @ChiefGokhlayehBosch (still can't add our GoD...).

Reply:
Imho, for an outsider first looking at Kiso, the Quick Start and other documentation resources are probably more relevant.

Reply:
we need to catch the reader with our first impression. Will he stay and identify with the content and purpose, is he/she actually the persona we are looking for?

Like the idea but am a bit concerned to lose all the readers looking for the quick start...

Reply:
Okay, I'm convinced. I'll remove the pull-request - should be put in Hugo.
@ChiefGokhlayehBosch if you have any free capacity I would appreciate your help here - I don't know when I can to this transfer towards Hugo.

Reply:
Ah and one more thing - I still believe we should add a "Scope" link in our resources :-)
Anyway, looking forward for the Kiso intro image from our GoD